features/checkout/paying_for_order/paying_with_paypal_during_checkout.feature

```diff
@@ -46,10 +49,10 @@ Feature: Paying with paypal during checkout
         Given I am logged in as "john@example.com"
         And I added product "PHP T-Shirt" to the cart
         And I proceed selecting "PayPal Express Checkout" payment method
-        And I confirm my order with paypal payment
+        And I confirm my order
```

Comment:
We should used a past tense to describe an initial state of system. `And I have confirmed my order`? Same for the rest of steps in `Given` block.